### ISSUE TITLE
Unable to tweak

### ISSUE THREAD

**Issue by EngineerXL** [micdoodle8/Galacticraft#3514]
**Unable to remove Desh ingot from melting in furnace** 

Minecraft version: 1.12.2
Galacticraft version: 181

Single player (SSP), Multiplayer (SMP), or SSP opened to LAN (LAN)? - SSP and SMP

Galacticraft add-ons and other mods installed? - No

Please provide screenshots / crash report / log as needed.
Why is this problem on your side №1? - https://github.com/CraftTweaker/CraftTweaker/issues/602 
Why is this problem on your side №2? - https://github.com/MJRLegends/GalacticraftTweaker/issues/4 
CraftTweaker.log: https://pastebin.com/30BjQHR1
Script: https://pastebin.com/eK54gAFR

**Comment by IcedReaper** [comment]
I would really like to get this fixed too.
Thanks in advance.

**Comment by Robijnvogel** [comment]
```The inits of the recipes for the planets mars and venus are too late in the postInit event.```
That's what I don't get. In GCCore and in GCPlanets, these recipes are registered in virtually the same spot.
And looking at the dependencies of GCCore, GCPlanets and CraftTweaker, I can think of no reason why GCCore would be sorted before- and GCPlanets after CraftTweaker.

And yes, exactly as you say, this is a problem with all Mars and Venus ores. GCCore works fine. 

I was in the process of unifying furnace-smelting recipes when I ran into this. 
I also found that the smelting recipes for __Asteroid__ ores were successfully removed by my script. The new recipes, however, weren't registered. (I was using the ore-dictionary entries for the new recipes.)
